.github/pull_request_template.md

```diff
@@ -1,11 +1,41 @@
-<!-- What's this PR for?  (Just a link to an issue is fine.) -->
+<!-- Describe your pull request here.-->
 
-**Testing plan:** <!-- How have you tested? -->
+**Fixes:** <!-- Issue link, or clear description.-->
```

Comment:
GitHub does not recognize `**Fixes:** #12345` as an issue closer like it recognizes `Fixes: #12345` or `Fixes #12345`.

Reply:
That seems like a good reason to remove the bold, thanks for catching.